### PR TITLE
New version: Harmonoid.Harmonoid version 0.2.3

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Harmonoid |     |
| Version     | 0.2.3     |  |
| Publisher   | Hitesh Kumar Saini   |       |
| ProductCode |           |     |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/59755)

### CODE REVIEW

File: manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VC++2015-2022Redist-x64
+ReleaseDate: 2022-05-07
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/harmonoid/harmonoid/releases/download/v0.2.3/harmonoid-windows-setup.exe
+  InstallerSha256: 74536AC6FD82A75B467584F462B6265028FF9A120D6E39D533025DE480DE83C4
```
Comment (alexmercerind) [inline manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.installer.yaml:21]: May you please update this to `19045770E06B81A00BB9D50B113FA521A32BBD791139697DE4F569464D7F1B15`.

Thanks!

**Review by alexmercerind — CHANGES_REQUESTED** [review]

File: manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [inline manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.installer.yaml:21]:
```suggestion
  InstallerSha256: 19045770E06B81A00BB9D50B113FA521A32BBD791139697DE4F569464D7F1B15
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Harmonoid.Harmonoid
+PackageVersion: 0.2.3
```
Comment (ItzLevvie) [inline manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.installer.yaml:5]:
```suggestion
PackageVersion: 0.2.3.0
```

File: manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Harmonoid.Harmonoid
+PackageVersion: 0.2.3
```
Comment (ItzLevvie) [inline manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.2.3.0
```

File: manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Harmonoid.Harmonoid
+PackageVersion: 0.2.3
```
Comment (ItzLevvie) [inline manifests/h/Harmonoid/Harmonoid/0.2.3/Harmonoid.Harmonoid.yaml:5]:
```suggestion
PackageVersion: 0.2.3.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by alexmercerind** [comment]
Thankyou!

**Pull request merged**